### ISSUE TITLE
NTR: Skin: Auspitz sign, positive (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8034]
New term request Skin: Auspitz sign, positive (seidel_190609200427): When scale is removed, pinpoint bleeding occurs; psoriasis  UMLS: 0277962

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000724
- label: Auspitz sign
- parents: [Abnormality of skin physiology (HP:0011122)](https://hpo.jax.org/app/browse/term/HP:0011122)
- pmid: PMID:2370333
- def: The appearance of small bleeding points after successive layers of scale have been removed from the surface of psoriatic papules or plaques.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8034
- synonyms:
